client/utils/rest.go

```diff
@@ -116,12 +117,13 @@ type BaseReq struct {
 
 // NewBaseReq creates a new basic request instance and sanitizes its values
 func NewBaseReq(
-	name, password, memo, chainID string, gas, gasAdjustment string,
+	name, password, from, memo, chainID string, gas, gasAdjustment string,
 	accNumber, seq uint64, fees sdk.Coins, genOnly, simulate bool) BaseReq {
 
 	return BaseReq{
 		Name:          strings.TrimSpace(name),
 		Password:      password,
+		From:          strings.TrimSpace(from),
```

Comment:
from should be an `sdk.AccAddress` to avoid checking if the string is a valid address